### ISSUE TITLE
[Feature Request] Select multiple frames

### ISSUE THREAD

**Issue by dahliadahlia** [piskelapp/piskel#766]
Hello Piskel community!

I don't know if this is possible, I haven't been able to figure it out. But I think it'd be really useful to be able to select multiple frames. I keep running into the situation where I have to create several copies of frames, or move more than one frame at a time.

Being able to select multiple frames would allow users to copy / move / delete multiple frames at once.

Thanks for reading :)

**Comment by carlsmith** [comment]
It sounds like it'd be useful, but *might* also complicate the Layers Window unacceptably.

Julian has done a really clever job of managing the compromises in interface design to create something fun and intuitive that is still capable enough to do serious work. New features shouldn't spoil that.

I can't speak for other people, so can't say how much it difference it would ultimately make, but if you could explain the details (maybe provide a rough visual mockup) of how this would work, it would at least help everyone understand what you are proposing.

**Comment by AthosTaz** [comment]
I agree with dahliadahlia! It would be so much easier to select multiple frames to copy/move them all at once instead copying/moving one by one.
Thanks! :)

**Comment by OPStellar** [comment]
In that case, it would be nice to select multiple frames, but that would require a primary frame to remain in focus. Say you have a selection copied and dragged, and then you select multiple frames. What will happen to the original selection? Is it lost? Does the display even show? Be specific with your expectations and requests, or else you might be asking for a great deal of advanced programming without realizing it.

**Comment by AthosTaz** [comment]
Yeah, I don't really know how difficult it would be to implement this, but what I imagined was the behaviour to be something like, while you hold let's say Ctrl, you multiselect. 
So, if you have a selection copied and dragged, and then you select multiple frames while holding Ctrl, the original selection remais selected. If not holding Ctrl, the original selection is lost. 
About the frame in focus it could be the first one selected...but again, I don't if it's possible and/or difficult. I just started using Piskel and try to multiselect with Ctrl or Shift by instinct and missed this command. 
But nevertheless, I'm finding Piskel great so far! :)

**Comment by OPStellar** [comment]
MULTISELECTION! Now THAT is a useful feature! If we could select multiple items on the canvas, that could segue into selecting multiple frames once a feature like that has been tested out.
So, going one step at a time, I really like the idea of a multi-selection option.
Great suggestion!

**Comment by zrulldan** [comment]
2025, still missing this feature. Outrageous that your options for exporting are either every frame or one at a time. But apparently this is impossible to fit in with layers for some(?) reason.
